src/make.ts

```diff
@@ -456,15 +459,16 @@ export async function generateParseContent(progress: vscode.Progress<{}>,
         }
         dryrunFile = util.resolvePathToRoot(dryrunFile);
         logger.message(`Writing the dry-run output: ${dryrunFile}`);
-        util.writeFile(dryrunFile, configuration.getConfigurationMakeCommand() + " " + makeArgs.join(" ") + "\r\n");
+        const lineEnding: string = (process.platform === "win32" && process.env.MSYSTEM === undefined) ? "\r\n" : "\n";
```

Comment:
Should we just make life easier on ourselves and only use `\n` line endings?

Reply:
Yes I ended up doing that. Plus some extra \r\n ==> \n replacements in baseline and runtime logs, because still some were created by various string operations or... I didn't find exactly what. The replacements are also helping in future for when I update the baselines, so that I copy from whenever without impacting the comparison.